### ISSUE TITLE
Unity SDK v6: Non-Organic tracking does not cause non-organic 

### ISSUE THREAD

**Issue by MagicMiikka** [AppsFlyerSDK/appsflyer-unity-plugin#34]
I was integrating the Unity SDK v6.0.1 Beta and noticed that non-organic tests were returning organic as the source when following these instructions:
https://support.appsflyer.com/hc/en-us/articles/360011451918-iOS-SDK-V6-beta-integration-guide-for-developers#integration-43-simulate-a-nonorganic-install
The clicks were properly registered and I can see it in my dashboard.
I Installed the SDK patch build via XCode as per instructions on production configurations.

Tested on XCode 11.6 & XCode 12.0 Beta 3 and neither properly registered the user as non-organic.
There are no Unity specific instructions atm.

Side note, checked for this as well but the prompt still did not trigger, nor did my install trigger as non-organic. https://github.com/AppsFlyerSDK/appsflyer-unity-plugin/issues/31

Device had iOS 14 Public Beta

**Comment by wesfieldj** [comment]
@MagicMiikka 

Can you run these steps:
1. Uninstall the app from the test device 
2. Make sure test device is [whitelisted](https://support.appsflyer.com/hc/en-us/articles/207031996#add-a-device-manually-via-the-user-interface) 
3. Click on an AppsFlyer link on the test device 
4. Re-install the app on test device


The testing should be the same on Unity. There is no need for Unity specific instructions. 

If your device is registered as a test device then running the steps above should result in a non-organic install.

**Comment by MagicMiikka** [comment]
I did this procedure multiple times yesterday.
Checked today and DeviceID said the device is already whitelisted as expected.

The clicks on the link are registered properly
I can after the Xcode install that it's a first open but organic.
The only thing I noticed that there is no prompt for tracking (same as #31) (DeviceID app requested this nicely)

Is there missing documentation on how to trigger the request when there is the Info.plist entry?
